src/main/kotlin/inc/evil/d17map/controllers/ExtraRoomsController.kt

```diff
@@ -13,18 +13,23 @@ import org.springframework.web.bind.annotation.*
 import java.util.*
 
 @RestController
-@RequestMapping("/extra-rooms")
 @Tag(name = "Extra Rooms")
 class ExtraRoomController(private val extraRoomService: ExtraRoomService) {
 
+    companion object {
```

Comment:
W sumie można by zrobić jakąś klasę z endpointami, żeby się nie powtarzać, ale to ogarniemy już po spotkaniu piątkowym podczas refactoringu